packages/contracts-bedrock/scripts/Deployer.sol

```diff
@@ -464,7 +530,9 @@ abstract contract Deployer is Script {
         if (_isProxy) {
             _contractName = string.concat(_contractName, "Proxy");
         }
-        bytes32 slotVal = vm.load(mustGetAddress(_contractName), bytes32(vm.parseUint(slot.slot)));
+        address addr = getAddress(_contractName);
```

Comment:
Now that `mustGetAddress` is no longer used here, could we accidentally load an empty slot value later?